docs/plugins/convert.rst

```diff
@@ -62,6 +62,9 @@ file. The available options are:
   Default: none (system default),
 - **copy_album_art**: Copy album art when copying or transcoding albums matched
   using the ``-a`` option. Default: ``no``.
+- **copy_album_art_maxwidth**: Downscale album art if it's too big. The
+  resize operation reduces image width to at most ``maxwidth`` pixels. The
+  height is recomputed so that the aspect ratio is preserved.
```

Comment:
Thanks for adding the docs! :sparkles: Would you mind also adding a bullet to the changelog?